src/styles/blocks/_container.scss

```diff
@@ -0,0 +1,79 @@
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
```

Comment:
Do non set margin for block. You could use mix.
![Screenshot 2022-08-02 at 01 42 37](https://user-images.githubusercontent.com/17098736/182257635-9eb7e2d5-af1d-456d-9d27-5d432470a548.png)